dask_expr/io/csv.py

```diff
@@ -1,16 +1,27 @@
 import functools
+import operator
 
+from dask_expr._util import _convert_to_list
 from dask_expr.io.io import BlockwiseIO, PartitionsFiltered
 
 
 class ReadCSV(PartitionsFiltered, BlockwiseIO):
-    _parameters = ["filename", "usecols", "header", "_partitions", "storage_options"]
+    _parameters = [
+        "filename",
+        "columns",
+        "header",
+        "_partitions",
+        "storage_options",
+        "_series",
+    ]
     _defaults = {
-        "usecols": None,
+        "columns": None,
         "header": "infer",
         "_partitions": None,
         "storage_options": None,
+        "_series": False,
     }
+    _absorb_projections = False
```

Comment:
This is a TODO for the future?

Reply:
Yeah, forgot about this. Just submitted https://github.com/dask-contrib/dask-expr/pull/268